modules/nw-egress-router-http-proxy-mode.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_sdn/deploying-egress-router-http-redirection.adoc
+
+[id="nw-egress-router-http-proxy-mode_{context}"]
+= Deploying an egress router pod in HTTP proxy mode
+
+In _HTTP proxy mode_, an egress router pod runs as an HTTP proxy on port `8080`. This only works for clients connecting to HTTP or HTTPS-based services, but usually requires fewer changes to the client pods. Applications must be configured to use an HTTP proxy by setting an environment variable.
```

Comment:
Same comment as in the other file that uses this content

Reply:
@jeana-redhat, I hadn't realized this is just a duplicate of the other content. Oh well.